app/pages/settings/index.cjsx

```diff
@@ -20,6 +21,7 @@ UserSettingsPage = React.createClass
 
   render: ->
     <section className="user-profile-content">
+      <Helmet title="#{counterpart 'userSettingsPage.title'} » #{@props.user.display_name}" />
       <div className="secondary-page settings-page">
         <h2><Translate content="userSettingsPage.header" /></h2>
```

Comment:
Changing the counterpart object broke this header: 
<img width="842" alt="screen shot 2017-07-24 at 11 57 49 am" src="https://user-images.githubusercontent.com/5016731/28534759-6df290da-7067-11e7-932a-10a3f318f38c.png">